include/deal.II/grid/grid_out.h

```diff
@@ -1080,6 +1080,25 @@ class GridOut
                   std::ostream                      &out) const;
 
   /**
+   * Write triangulation in VTU format for each processor, then combine these
+   * files into a .pvtu file. The input <tt>view_levels</tt> is set to false by
+   * default, allowing only the active level cells to be written into the .vtu
+   * file, but can be set as true to view each level of a multilevel method.
+   * The input <tt>include_artificial</tt> is also set to false by default,
+   * allowing the processor to only write in owned and ghost cells into its .vtu
+   * file, but can be set as true to view the artificial cells for each processor.
+   * Each .vtu and .pvtu file will have the attributes subdomain, lvlsubdomain, level,
+   * and proc_writing. The level value can be used to seperate the image into the
+   * view of the gird on each level of a multilevel method and the proc_writing value
```

Comment:
please add a short sentence for each parameter in the following format:
```

@param tria The Triangulation that should be written to the file.

@param filename ...
```

See ``integrate_difference`` for an example: https://github.com/tcclevenger/dealii/blob/write_parallel_vtu/include/deal.II/numerics/vector_tools.h#L1848